Tests/SonarScanner.MSBuild.PreProcessor.Test/AnalysisConfigProcessing/AnalysisConfigGeneratorTests.cs

```diff
@@ -545,28 +538,29 @@ public void GenerateFile_SourcesTestsIgnored(string sources, string tests)
         config.LocalSettings.Should().NotContain(x => x.Id == "sonar.tests");
     }
 
-    [TestCategory(TestCategories.NoUnixNeedsReview)]
     [TestMethod]
     public void GenerateFile_TrustStoreProperties_Mapped()
     {
+        var pfxPAth = "\"C:/path/to/truststore.pfx\"";
```

Comment:
This path should be also with Windows separator.
```suggestion
        var pfxPAth = "\"C:\\path\\to\\truststore.pfx\"";
```

That means that the assert should be updated as well as the expect value is `"\"C:/path/to/truststore.pfx\""`